packages/wonder-blocks-icon-button/src/components/activity-icon-button.tsx

```diff
@@ -0,0 +1,261 @@
+            borderRadius: theme.root.border.radius,
+            color: themeVariant.foreground[kind].rest,
+            // layout
+            paddingBlockEnd: theme.root.shadow.y.rest,
+            flexDirection: "column",
+            // Prevent the button from stretching to fill the parent
+            alignSelf: "center",
+            justifySelf: "center",
+            /**
+             * States
+             *
+             * Defined in the following order: hover, active, focus.
+             *
+             * This is important as we want to give more priority to the
+             * :focus-visible styles.
+             */
+            ":hover": {
+                color: themeVariant.foreground[kind].hover,
+                // paddingBlockEnd: `calc(${theme.root.shadow.default.y} * 2)`,
+            },
+            [":is(:hover) .chonky" as any]: {
+                background: themeVariant.background[kind].hover,
+                border: `${borderWidthKind.hover} solid ${themeVariant.border[kind].hover}`,
+                boxShadow: `0 ${theme.root.shadow.y.hover} 0 0 ${themeVariant.shadow[kind].hover}`,
+                // motion
+                transform: `translateY(calc((${theme.root.shadow.y.hover} - ${theme.root.shadow.y.rest}) * -1))`,
+                // transform: `translateY(-2px)`,
+            },
+
+            [":is(:active) .chonky" as any]: chonkyPressed,
+
+            // :focus-visible -> Provide focus styles for keyboard users only.
+            ...focusStyles.focus,
+        },
+        disabled: {
+            cursor: "not-allowed",
+            ...disabledStatesStyles,
+            // Reset hover and active styles on disabled buttons.
+            ":hover": disabledStatesStyles,
+            ":active": disabledStatesStyles,
+            ":focus-visible": {
+                transform: "none",
+            },
+            // Reset hover and active styles on the chonky element.
+            [":is(:hover) .chonky" as any]: disabledStatesStyles,
+            [":is(:hover) .chonky" as any]: chonkyDisabled,
+            [":is(:active) .chonky" as any]: chonkyDisabled,
+        },
+        // Enable keyboard support for press styles.
+        pressed: {
+            [".chonky" as any]: chonkyPressed,
+        },
+
+        chonky: {
+            // layout
+            // Used for the chonky box.
+            borderRadius: theme.root.border.radius,
+            paddingBlock: theme.root.layout.padding.block,
+            paddingInline: theme.root.layout.padding.inline,
+            // theming
+            background: themeVariant.background[kind].rest,
+            border: `${borderWidthKind.rest} solid ${themeVariant.border[kind].rest}`,
+            // Gives the button a "chonky" look and feel.
+            boxShadow: `0 ${theme.root.shadow.y.rest} 0 0 ${themeVariant.shadow[kind].rest}`,
+            // motion
+            transition: "all 0.12s ease-out",
+
+            ["@media not (hover: hover)" as any]: {
```

Comment:
I was curious about how it looks in mobile so I tried it out! I found that after clicking one chonky button, when I click another chonky button, the first button de-presses. Is that intended?

https://github.com/user-attachments/assets/93d047a9-d852-47cc-b611-50dcb5cd9641